apps/rush-lib/src/data/RushHooks.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { IRushHooksJson } from './RushConfiguration';
+
+/**
+ * Rush hook names
+ * @alpha
+ */
+export enum RushHookName {
+  /**
+   * Hook name for post every rush command.
+   */
+  postBuild = 1
+}
+
+/**
+ * This class represents rush hooks configured for this repo.
```

Comment:
Also maybe give a little more detail:

"Hooks are user-defined script actions that Rush executes when specific events occur.  The actions are expressed as a command-line that is executed using the operating system shell."

---
In reply to: [117120289](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117120289) [](ancestors = 117120289)